### PR TITLE
[Settings] Kotlinize PrivacyPolicyActivity

### PR DESCRIPTION
**Pull request by JuancaG05**
Implements https://github.com/owncloud/android/issues/3129

### CODE REVIEW

File: owncloudApp/src/main/java/com/owncloud/android/presentation/ui/settings/SettingsActivity.kt [hunk 1]
```diff
@@ -34,8 +36,12 @@ class SettingsActivity : AppCompatActivity() {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_settings)
 
-        val toolbar = findViewById<Toolbar>(R.id.toolbar).apply {
+        val toolbar = findViewById<Toolbar>(R.id.standard_toolbar).apply {
             setTitle(R.string.actionbar_settings)
+            visibility = View.VISIBLE
```
Comment (abelgardep) [inline owncloudApp/src/main/java/com/owncloud/android/presentation/ui/settings/SettingsActivity.kt:41]: Same here
```suggestion
            isVisible = true
```

**Review by abelgardep — CHANGES_REQUESTED** [review]
Some changes here @JuancaG05

File: owncloudApp/src/androidTest/java/com/owncloud/android/settings/more/PrivacyPolicyActivityTest.kt [hunk 2]
```diff
@@ -59,6 +62,11 @@ class OCSettingsPrivacyPolicyTest {
         onView(withText(R.string.actionbar_privacy_policy)).check(matches(isDisplayed()))
     }
 
+    @Test
+    fun checkProgressbar() {
+        onView(withId(R.id.syncProgressBar)).check(matches(isDisplayed()))
```
[the remark this answers is not in the document]
Comment (JuancaG05) [reply owncloudApp/src/androidTest/java/com/owncloud/android/settings/more/PrivacyPolicyActivityTest.kt:67]: Didn't know about that, very interesting and simple to use, will take into account from now on!

**Review by abelgardep — APPROVED** [review]
Looks good to me! Good job 💯

**Pull request merged**